projects/dsp-ui/src/lib/viewer/values/time-value/time-input/time-input.component.ts

```diff
@@ -211,6 +246,8 @@ export class TimeInputComponent extends _MatInputMixinBase implements ControlVal
     }
 
     _handleInput(): void {
+        this.dateFormControl.updateValueAndValidity();
```

Comment:
Why is this needed here?

Reply:
If we don't, `dateTimeValidator` is not triggered and the error message from it remains.
<img width="875" alt="Screen Shot 2021-01-14 at 10 42 33" src="https://user-images.githubusercontent.com/60604010/104574254-1ab24900-5656-11eb-989b-51f2f9583dea.png">

Reply:
Is this also the case in the other components like date and interval? Or is there something special about time?

Reply:
Nothing special about time (this time), we do it in the Date and Interval value components as well.